flare/model_zoo/distributional_rl_models.py

```diff
@@ -0,0 +1,72 @@
+from simple_models import SimpleModelQ
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import math
+
+
+class SimpleModelC51(SimpleModelQ):
+    def __init__(self, dims, num_actions, mlp, vmax, vmin, bins):
```

Comment:
change 'mlp' to 'perception_net' because potentially it can be a CNN or other network.